app-misc/nnn/nnn-4.7.ebuild

```diff
@@ -0,0 +1,87 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit bash-completion-r1 flag-o-matic toolchain-funcs xdg-utils
+
+DESCRIPTION="The missing terminal file browser for X"
+HOMEPAGE="https://github.com/jarun/nnn"
+SRC_URI="https://github.com/jarun/nnn/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+IUSE="+readline pcre 8contexts icons nerdfonts emoji qsort"
+
+DEPEND="sys-libs/ncurses:=
+	pcre? ( dev-libs/libpcre )
+	readline? ( sys-libs/readline:= )
+	elibc_musl? ( sys-libs/fts-standalone )"
+BDEPEND="virtual/pkgconfig"
+RDEPEND="${DEPEND}"
+REQUIRED_USE="?? ( icons nerdfonts emoji )"
+
+src_prepare() {
+	default
+	tc-export CC
+	use elibc_musl && append-flags "-lfts"
+	sed -i -e '/install: all/install:/' Makefile || die "sed failed"
+}
+
+src_compile() {
+	nnn_opts=(
+		O_NORL=$(usex readline 0 1)
+		O_PCRE=$(usex pcre 1 0)
+		O_CTX8=$(usex 8contexts 1 0)
+		O_ICONS=$(usex icons 1 0)
+		O_NERD=$(usex nerdfonts 1 0)
+		O_EMOJI=$(usex emoji 1 0)
+		O_QSORT=$(usex qsort 1 0)
+	)
+	emake "${nnn_opts[@]}"
+}
+
+src_install() {
+	emake PREFIX="${EPREFIX}/usr" DESTDIR="${D}" install
+
+	emake PREFIX="${EPREFIX}/usr" DESTDIR="${D}" install-desktop
+
+	newbashcomp misc/auto-completion/bash/nnn-completion.bash nnn
+
+	insinto /usr/share/fish/vendor_completions.d
+	doins misc/auto-completion/fish/nnn.fish
+
+	insinto /usr/share/zsh/site-functions
+	doins misc/auto-completion/zsh/_nnn
+
+	einstalldocs
+
+	insinto /usr/share/nnn
+	doins -r plugins
+}
+
+pkg_postinst() {
+	xdg_desktop_database_update
+
+	elog "nnn plugins are installed to /usr/share/nnn/plugins/, but nnn does not"
+	elog "load them fom this directory. You will need to copy/symlink them to"
+	elog "~/.config/nnn/plugins/ if you want to use them."
+	elog "Note that some plugins have runtime dependencies that may need to be installed."
+	elog "Refer to the individual plugin's in-file documentation for more information."
+
+	if use icons; then
+		einfo "In order for file icons to work, your terminal needs to use icons-in-terminal."
+		einfo "See https://github.com/sebastiencs/icons-in-terminal"
+	elif use nerdfonts; then
+		einfo "In order for file icons to work, your terminal needs to use a patched nerdfont."
+		einfo "See https://www.nerdfonts.com/"
+	elif use emoji; then
+		einfo "In order for file icons to work, your terminal needs to use a font that"
+		einfo "includes standard unicode emoji."
+	fi
```

Comment:
I wonder if we're better switching to `elog` here. IIRC `einfo` doesn't show if you specify `--jobs` for your emerge.